src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/entities/ccd/CaseDetails.java

```diff
@@ -47,10 +47,6 @@ public String getJurisdiction() {
 
     public State getState() {
 
-        if (state == null) {
```

Comment:
Why did we remove these?

Reply:
This validation happens before ccd assigns state to a case and hence this condition had to be taken out. Discussed this with @Marcin and we checked with you as well during implementation.